corehq/apps/data_interfaces/models.py

```diff
@@ -879,7 +879,6 @@ def when_case_matches(self, case, rule):
                 continue
             result = update_case(case.domain, case_id, case_properties=properties, close=False,
                 xmlns=AUTO_UPDATE_XMLNS)
-
```

Comment:
nit: unnecessary change in an otherwise unchanged file

Reply:
oops